pom.xml

```diff
@@ -57,6 +57,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Are these changes relevant to the `apns-collapse-id` header? Can we do without them?